addons/bestja_project_hierarchy/models.py

```diff
@@ -43,6 +43,7 @@ class ProjectInvitation(models.Model):
         compute_sudo=True,
     )
     invited_projects = fields.One2many('bestja.project', inverse_name='parent_invitation')
+    inviter = fields.Boolean(string="Zapraszający", compute='_compute_inviter')
```

Comment:
It is weird, indeed, however when you try to accept it (being the inviter) you get a message that you're not a coordinator of the proper organization - so I made the condition similar.